contents/blog/best-mixpanel-alternatives.md

```diff
@@ -0,0 +1,498 @@
+ <td><strong>Open source</strong></td>
+ <td className="text-center"><span className="text-green text-lg">✔</span></td> 
+ <td className="text-center"><span className="text-red text-lg">✖</span></td>
+ <td>Build your own apps and contribute code</td>
+ </tr>
+ <tr>
+ <td><strong>Autocapture</strong></td>
+ <td className="text-center"><span className="text-green text-lg">✔</span></td> 
+ <td className="text-center"><span className="text-red text-lg">✖</span></td>
+ <td>Capture events without manual instrumentation</td>
+ </tr>
+ <tr>
+ <td><strong>SQL insight builder</strong></td>
+ <td className="text-center"><span className="text-green text-lg">✔</span></td> 
+ <td className="text-center"><span className="text-red text-lg">✖</span></td>
+ <td>Write your own insights using SQL</td>
+ </tr> 
+ <tr>
+ <td><strong>Notebooks</strong></td>
+ <td className="text-center">Public beta</td> 
+ <td className="text-center"><span className="text-green text-lg">✔</span></td>
+ <td>Collaborate on analysis in shareable notebooks</td>
+ </tr> 
+ </tbody>
+</table>
+</div>
+
+![posthog](../images/blog/mixpanel-alternatives/posthog.png)
+
+### How popular is PostHog?
+
+According to [data compiled by Jason Packer](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/), an independent analytics consultant, PostHog is the fastest growing Mixpanel alternative in the market.
+
+As of July 2023, **PostHog** is deployed by 2,365 (0.2%) of the top 1 million websites in 2023, up +327% from 554 the previous year. **Mixpanel** is deployed by 5,218 (0.5%), down -17% from 6,295 the previous year.
+
+### Why do companies use PostHog?
+
+According to [reviews on G2](https://www.g2.com/products/posthog/reviews), companies use PostHog because:
+
+1. **It replaces multiple tools:** PostHog can replace Mixpanel (product analytics), LaunchDarkly (feature flags and A/B testing), and Hotjar (session replay and surveys). This simplifies workflows and ensures all their data is in one place.
+
+2. **Pricing is transparent and scalable:** Reviewers appreciate how PostHog's pricing scales as they grow. There's a [generous free tier](/pricing). Companies eligible for [PostHog for Startups](/startups) also get $50k in additional free credits.
+
+3. **They need a complete picture of users:** PostHog includes every tool necessary to understand users and improve products. This means creating funnels to track conversion, watching replays to see where users get stuck, testing solutions with A/B tests, and gathering feedback with user surveys.
+
+> ### Bottom line 
+>
+> PostHog is the best Mixpanel alternative for startups and mid-size companies. It replaces Mixpanel and numerous other tools, saving money and time. Power user features, like an SQL insight builder and console logs, make it a good choice for engineering-led teams, too.
+
+## Google Analytics 4 (GA4)
+
+![GA4](../images/blog/mixpanel-alternatives/google-website.png)
```

Comment:
This screenshot made me chuckle

Reply:
Haha, yes. I liked your website screenshots in the vs you wrote. Figured I'd let their marketing do the talking ;)